### DISCUSSION TITLE
ConnectedDriive subscription required? Any possible work around?

### DISCUSSION THREAD

**Discussion by sxjuan** [BimmerGestalt/AAIdrive#629] [General]
My $50 base subscription just ended, and when I go to the connectedDrive store, wow the only option I have is the $120 combo package... This AAIdrive app was the only reason that I went for the base subscription but now they bumped the cost to $120, so is there any work around that we can still use the app protocol without an active subscription?

I believe this should be an ongoing issue for now as at some point it will be so expensive to use. I have also heard that they shut down the connectedDrive features for some models due to 3G network shutoff.

Thanks in advance

**Comment by sxjuan** [comment]
UPDATE: I have also tried downloading and installing the old Connected Android app but nothing seems to be happening positively. So do we need FSC coding for this? Any guidelines?

**Comment by hufman** [comment]
I don't believe there is any way for an app like AAIdrive to enable such a feature in the car. Besides the subscription, I suspect there's probably some coding way to do it, because Minis come with the feature from the factory without a subscription. Unfortunately, I don't know enough about coding to help there.

**Comment by nazeboy** [comment]
I am able to get the app running, however, when I run Spotify and the car is in motion the added features of Spotify stop working. Is there a workaround?

**Reply by hufman** [reply]
That sounds like the Australian app restriction, which I haven't heard of any workarounds for.

**Comment by nazeboy** [comment]
Hi Walter,

Hope you are well. Are you saying it is the iDrive 5 issue and that there is no fix to stop it from switching off applications once in motion?

Is this something the team will add for users in Australia on the AAIdrive app?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by hufman** [reply]
I don't know much about the Australian limit, and my quick search doesn't show any easy coding fixes for it, so you're on your own if you want to figure it out. There's no way AAIdrive can work around this limit in the IDrive, unfortunately.

**Comment by nazeboy** [comment]
Hi Walter,  This is to do with Australian Design Rules (ADR - https://www.infrastructure.gov.au/infrastructure-transport-vehicles/vehicles/vehicle-design-regulation/australian-design-rules ), which unfortunately it looks like BMW at the time took to the extreme and switched of the capability whilst the vehicle is in motion to be able to let Spotify work in iDrive 6 and below.

It would no doubt require a hack into the code on the iDrive.

Thanks for your help.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
